src/modules/Packages.rb

```diff
@@ -29,6 +29,13 @@ class PackagesClass < Module
     # Minimum set of packages required for installation with remote X11 server
     REMOTE_X11_BASE_TAGS = [ "xorg-x11-server", "xorg-x11-fonts", "icewm" ]
 
+    # Some products are already be "included" in other products. So they MUST
+    # not be installed anymore because they are conflicting to the other one.
+    PRODUCT_CONFLICTS = {
+      # SLES_SAP includes SLES. So SLES is not needed anymore
+      "SLES" => [ "SLES_SAP" ]
```

Comment:
Maybe we should change the mapping direction? Because in reality it is *SLES_SAP* which conflicts with *SLES*.

The conflicts goes from *SLES_SAP* so logically it would be better to have `"SLES_SAP" => [ "SLES" ]` mapping. (And update the code respectively.)